### DISCUSSION TITLE
How to create custom 2d slice with formula combining with existing quantities

### DISCUSSION THREAD

**Discussion by AS-Proconsultiers** [firemodels/fds#13206] [Q&A]
Hi,
I need to create a custom 2d slice with a combination of existing quantities and mathematical formulas in pyrosim.

Eg. 
<img width="314" alt="OU formula" src="https://github.com/user-attachments/assets/f5f57386-ee82-43b0-8a21-7e8b66c522d1">

The above formula is for custom 2d slice. Please help me achieve this. Thanks in advance.

**Comment by TristanHehnen** [comment]
You could use the fdsreader Python module. It can read data, like `SLCF` or `BNDF`, from FDS into NumPy arrays and from there you can process it however you like:
https://firedynamics.github.io/LectureFireSimulation/content/tools/03_analysis/02_fdsreader.html
